src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbolBase.cs

```diff
@@ -36,8 +36,11 @@ internal abstract class SourceOrdinaryMethodSymbolBase : SourceOrdinaryMethodOrU
             bool isExtensionMethod,
             bool isReadOnly,
             bool hasBody,
+            bool isExpressionBodied,
             bool isNullableAnalysisEnabled,
-            BindingDiagnosticBag diagnostics) :
+            BindingDiagnosticBag diagnostics,
+            RefKind refKind = RefKind.None,
```

Comment:
>RefKind refKind = RefKind.None [](http://example.com/codeflow?start=12&length=30)

Optional parameters are bug prone, we prefer adding parameters as required and adjusting all call sites. That is especially useful for code reviews because reviewers are able to examine all affected call sites.

Reply:
I was preserving the pattern you guys have already in sveral places here (including for flags).  Where things like isMetadataVirtualIgnoringModifiers is already optional.  It's unclear which direction is better to go here, but my preerence would be consistency.  If you would prefer explicitness, then i think it would be better to be consistently uniformly.

Reply:
> It's unclear which direction is better to go here, but my preerence would be consistency

My request was not about any kind of consistency. I am asking to not add optional parameters in context of this PR. No reason to touch other parameters.

Reply:
Done.